packages/cli/bin/n8n

```diff
@@ -55,6 +55,17 @@ if (process.env.NODEJS_PREFER_IPV4 === 'true') {
 // More details: https://github.com/nodejs/node/issues/48145
 require('net').setDefaultAutoSelectFamily?.(false);
 
+// WebCrypto Polyfill for older versions of Node.js 18
+if (!globalThis.crypto?.getRandomValues) {
+	const picocolors = require('picocolors');
+	console.warn(
+		picocolors.red(`Your Node.js version does not support WebCrypto.
+Please upgrade to the latest version of Node.js ${major(nodeVersion)}.`),
+	);
+	const { webcrypto } = require('node:crypto');
+	globalThis.crypto = { ...webcrypto };
```

Comment:
To better understand, why do we copy it when creating a shim?

Reply:
honestly I don't recall what I was thinking when I woke up. I'll change it to 
```
globalThis.crypto = crypto;
```